Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -168,10 +176,20 @@ private class ParameterSetNames
         /// </para>
         /// </summary>
         [Parameter(Position = 2, Mandatory = true, ParameterSetName = ParameterSetNames.ContentsFromFile)]
+        [ValidateNotNullOrEmpty()]
```

Comment:
You can still supply something like an empty string to the parameter and that doesn't get checked by mandatory.